deploy/ansible/worker/geth_inventory.yml

```diff
@@ -0,0 +1,20 @@
+all:
+  children:
+    nucypher:
+      children:
+        mainnet:
+          children:
+            nodes:
+              vars:
+                network_name: "mainnet"
+                geth_options: "--mainnet"
+                geth_dir: '/home/nucypher/geth/.ethereum/mainnet/'
+                geth_container_geth_datadir: "/root/.ethereum/mainnet"
+                nucypher_container_geth_datadir: "/root/.local/share/geth/.ethereum/mainnet"
+                etherscan_domain: mainnet.etherscan.io
+                ansible_python_interpreter: /usr/bin/python3
+                ansible_connection: ssh
+                ansible_ssh_private_key_file: <PEM FILE HERE>
```

Comment:
These would be manually edited.  It might be possible to remove them and then specify those two options on the cli when running the `ansible-playbook`.  I'm not going to spend time on it right now but maybe the next time we need to build new geth nodes I'll take a crack at it.

Reply:
@derekpierre check out this PR.  I think it addresses/is relevant to your concern somewhat.